views/building-hours/all-building-hours.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import {
+  Text,
+} from 'react-native'
+
+import moment from 'moment-timezone'
+const CENTRAL_TZ = 'America/Winnipeg'
+
+export function allBuildingHours(info, style) {
+  let dayTimes = []
+  let current = moment().tz(CENTRAL_TZ)
+
+  for (let i = 0; i < 7; i++) {
+    let hoursString = ''
+    let day = current.format('dddd')
+    let d = current.format('ddd')
+
+    current = current.add(1, 'days')
+
+    let timesArray = info.times.hours[d]
+
+    if (timesArray) {
+      let open = moment(timesArray[0], 'H:mm').format('h:mm a')
+      let close = moment(timesArray[1], 'H:mm').format('h:mm a')
```

Comment:
Don't forget to instantiate these with the final `tz` parameter, too. See https://github.com/StoDevX/AAO-React-Native/blob/master/views/building-hours/get-building-hours.js#L20 for a reference?